### ISSUE TITLE
Update mentions of emulators

### ISSUE THREAD

**Issue by exelotl** [gbadev-org/tonc#125]
In the new setup chapter, I mentioned no$gba as an emulator with unique debugging features. Since then, [Mesen2](https://www.mesen.ca/) released with GBA support which I believe is a better recommendation (more accurate, cross-platform, has all the same debugging features, can load newer ELF files which no$gba can't).

Tonc also refers to VBA in a lot of places, which should probably be updated too.

**Comment by asiekierka** [comment]
I agree, though it might be a good idea to leave a passing mention for NO$GBA at least, as it should run better on very low-end devices.

In addition, while NO$GBA and Mesen2 have good low-level debuggers, mGBA has a GDB stub, which is much better for debugging higher-level C/C++ code.

The VBA references should go.
